plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1697,7 +1697,7 @@ read_only::get_producers_result
 read_only::get_producers( const read_only::get_producers_params& params, const fc::time_point& deadline ) const try {
    abi_def abi = eosio::chain_apis::get_abi(db, config::system_account_name);
    const auto table_type = get_table_type(abi, "producers"_n);
-   const abi_serializer abis{ std::move(abi), abi_serializer::create_yield_function( abi_serializer_max_time ) };
+   const abi_serializer abis{ abi_def(abi), abi_serializer::create_yield_function( abi_serializer_max_time ) };
```

Comment:
I don't think this change is right, should still be `std::move(abi)` since `abi_serializer::abi_serializer( abi_def abi, const yield_function_t& yield )` constructor now takes a value. With the current version it will make an unnecessary copy.

Reply:
Never mind I see where `abi` is used afterwards.

Reply:
`abi` is used on line 1747, so it can't be `std::move` here. I added the explicit copy to make it obvious that a copy is needed.

Reply:
Yes, it can't be `std::move(abi)`, but it should be `abi`, not `abi_def(abi)`. My original comment was wrong.

Reply:
You are right, the explicit copy is fine after all!